sklearn/inspection/_plot/partial_dependence.py

```diff
@@ -314,13 +339,26 @@ def convert_feature(fx):
                 "Each entry in features must be either an int, "
                 "a string, or an iterable of size at most 2."
             )
-        if kind != "average" and np.size(fxs) > 1:
-            raise ValueError(
-                "It is not possible to display individual effects for more "
-                f"than one feature at a time. Got: features={features}."
-            )
+        # store the information if 2-way PD was requested with ICE to later
+        # raise a warning
+        ice_for_two_way_pd.append(kind_plot != "average" and np.size(fxs) > 1)
+
         tmp_features.append(fxs)
 
+    if any(ice_for_two_way_pd):
+        # only warn when 1- and 2-way PD were requested
+        warnings.warn(
+            "You requested an ICE plot with 2-way feature interactions. "
+            "This is impossible to render. We will instead make a regular PD "
+            "plot ('average'). If you want finer control on how to specify "
+            "the kind of plot to render (ICE vs PD), you may pass a list of "
+            "strings to the kind parameter.",
+            UserWarning,
+        )
+        kind_ = [
+            "average" if forcing_average else kind_plot
+            for forcing_average, kind_plot in zip(ice_for_two_way_pd, kind_)
+        ]
```

Comment:
Can we error here and suggest what the list of strings should be? I prefer teaching the user what the correct thing to do is, then to magically fix it for them.

Reply:
I now raise an error and indeed provide both the values of `features` and `kind` to be passed to make the code works. Indeed, this is not a regression so we can raise an error.